### PR TITLE
[script.colorbox] 2.0.2

### PR DESCRIPTION
**Pull request by BADMS**
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.colorbox/addon.xml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 		<import addon="script.module.simplejson" version="2.0.10"/>
 		<import addon="script.module.pil" version="1.1.7"/>
 	</requires>
-    <extension point="xbmc.python.script"           library="default.py">
+    <extension point="xbmc.python.script" library="default.py">
         <provides>executable</provides>
     </extension>
 	<extension point="xbmc.addon.metadata">
+	    <summary lang="en">Image manipulator and colour matching tools</summary>
 		<language/>
```
Comment (enen92) [inline script.colorbox/addon.xml]: please remove the language tag (only applied to content plugins - video, audio, etc)

File: script.colorbox/addon.xml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 		<import addon="xbmc.python" version="2.25.0"/>
 		<import addon="script.module.simplejson" version="2.0.10"/>
 		<import addon="script.module.pil" version="1.1.7"/>
 	</requires>
-    <extension point="xbmc.python.script"           library="default.py">
+    <extension point="xbmc.python.script" library="default.py">
         <provides>executable</provides>
     </extension>
 	<extension point="xbmc.addon.metadata">
+	    <summary lang="en">Image manipulator and colour matching tools</summary>
```
Comment (enen92) [inline script.colorbox/addon.xml]: your summary is duplicated. Also please update the "en" to "en_GB" (this can be done on a later update though)

**Comment by enen92** [comment]
@BADMS I meant to remove the `<language/>` entry on addon.xml. The summary should have a lang attribute like before but use "en_GB" as the value

**Comment by BADMS** [comment]
Ok, how about now?

**Comment by enen92** [comment]
thanks for adjusting @BADMS 
merged

**Comment by BADMS** [comment]
Thanks for help

_Sent from my Motorola XT1650 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Pull request merged**